lib/components/T3CeFeloginLogin/T3CeFeloginLogin.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="fe-login">
+    <p>{{ $props.data.message.header }}</p>
+    <p>{{ $props.data.message.message }}</p>
+    <T3Form ref="form" :elements="elements" @submit="onSubmit" />
+  </div>
+</template>
+<script>
+import T3Form from '../../components/T3Form/T3Form'
+import baseCe from '../../mixins/component/baseCe'
+
+export default {
+  name: 'T3CeFeloginLogin',
+  components: {
+    T3Form
+  },
+  extends: baseCe,
+  props: {
+    data: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    redirectUrl () {
+      const element = this.data.form.elements.filter(function (element) {
+        return element.name === 'redirect_url'
+      })
+
+      if (element.length > 0) {
+        return element?.value || ''
+      }
+
+      return ''
+    },
+    elements () {
+      return this.compileFormElements(this.data.form.elements)
+    }
+  },
+  methods: {
+    compileFormElements (elements) {
+      return elements.map((element) => {
+        return {
```

Comment:
we should remove empty submit
<img width="623" alt="Zrzut ekranu 2021-12-17 o 11 28 25" src="https://user-images.githubusercontent.com/7935392/146530571-4b76edaa-bb76-4623-be89-a60c48ae9bb8.png">
 here 
because w're using t3form cta

Reply:
Done ✅ 
every `type === 'submit'` gets filtered out before converting the element format.